.github/CONTRIBUTING.md

```diff
@@ -142,7 +149,16 @@ builder failures, you may be experiencing a [test suite
 issue](https://github.com/zfsonlinux/zfs/issues?q=is%3Aissue+is%3Aopen+label%3A%22Test+Suite%22).
 There are also various [buildbot options](https://github.com/zfsonlinux/zfs/wiki/Buildbot-Options)
 to control how changes are tested.
-* All proposed changes must be approved by a ZFS on Linux organization member.
+* If you have an idea you'd like to discuss or which requires additional testing consider opening it as a draft PR. 
+Once everything is in good shape and the details have been worked out you can remove its draft status. 
+Any required reviews can then be finalized and the PR merged.
+* Every PR will by tested by the CI on multiple platforms by running the [zfs-tests.sh](
```

Comment:
OK, just two last nits.

* Can you reorder this list so the last 3 bulleted items that we just added appear first in the "Tests and Benchmarks" list.  While everything in the list is important, these are what gets asked about the most.  

* Reading through the other items we always spell out "pull request" instead of "PR", and use "buildbot" instead of "CI".  To be consistent and clear, let's always use "pull request" and "buildbot".

Reply:
No problem, Nits and feedback exists for a reason...
There you go, done!